cmd/community_images/cli/root.go

```diff
@@ -143,7 +143,7 @@ func runPrettyCommand(v *viper.Viper) error {
 
 	config, _ := KubernetesConfigFlags.ToRESTConfig()
 	log.Header(headerLine(config.Host))
-	re := regexp.MustCompile(`^k8s\.gcr\.io/|^gcr\.io/google-containers`)
+	re := regexp.MustCompile(`^k8s\.gcr\.io/|^gcr\.io/google[-_]containers`)
```

Comment:
let me add a test case for this

Reply:
ah ... we don't have any for this code?

.. in that case, maybe later, this is a quick fix in aside to other work ...